lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -1985,7 +2025,8 @@ public List<String> listValues(TaskQueryColumnName columnName, SortDirection sor
     try {
       taskanaEngine.openConnection();
       this.columnName = columnName;
-      this.orderBy.clear();
+      this.orderByOuter.clear();
+      this.orderByInner.clear();
```

Comment:
Is this bevahiour documented?
e.g. someone could save the TaskQuery and execute 2 calls (listValues and list) and this behaviour would reset both order statements.

Either document that or restore the values after the data was fetched (and test that)